src/app/pages/test-centre-journal/components/candidate-search-card/candidate-search-card.ts

```diff
@@ -12,14 +18,34 @@ type CandidateData = {
   templateUrl: 'candidate-search-card.html',
   styleUrls: ['candidate-search-card.scss'],
 })
-export class CandidateSearchCardComponent {
+export class CandidateSearchCardComponent implements OnChanges {
```

Comment:
need to be using the DateTime class throughout this component instead of JS Date